jobs/ci-run/functional/amd64/bootstrap-constraints.yml

```diff
@@ -13,6 +13,10 @@
         default: 'amd64'
         description: 'Arch to build for.'
         name: BUILD_ARCH
+    - string:
+        default: jammy
+        description: ''
+        name: series
```

Comment:
job params are UPPER_CASE

Reply:
The lower case `series` parameter is passed from the root CI master to all the child jobs.
I can have a following PR the change it to UPPERCASE.